server-code/Projects.js

```diff
@@ -2,12 +2,48 @@
 Document: JS code specific for projects
 */
 
+const slideDeckStr = "Slide Deck";
```

Comment:
I love this! we haven't specifically called out constants in other areas. just two things:
1. let's use standard naming scheme for constants of something like `SLIDE_DECK_STR`
2. let's add a single line comment or something above to help explain what they are

Reply:
Resolved by commit b8d322e26867e709212c3ada222fd6fc394bde42